var/spack/repos/builtin/packages/petsc/package.py

```diff
@@ -704,6 +704,8 @@ def configure_options(self):
                 hip_ipkgs.extend(["rocrand"])
             else:
                 hip_lpkgs.extend(["rocrand"])
+            if spec.satisfies("^hipblas-common@6.3.0:"):
+                hip_ipkgs.extend(["hipblas-common"])
```

Comment:
Not clear to me where a dependency on `hipblas-common` is declared for `petsc`:
```
$ spack dependents hipblas-common
hipblas  hipblaslt
```
Should we check on `hipblas` instead? Also, not clear why there's a version bound here.

Reply:
```suggestion
            if spec.satisfies("^hipblas@6.3.0:"):
                hip_ipkgs.extend(["hipblas-common"])
```